Index: venv/lesson-2/task-7-additinally.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/venv/lesson-2/task-7-additinally.py b/venv/lesson-2/task-7-additinally.py
--- a/venv/lesson-2/task-7-additinally.py	
+++ b/venv/lesson-2/task-7-additinally.py	
@@ -1,3 +1,18 @@
 # Напишите программу, которая принимает на вход число N и выдает набор произведений чисел от 1 до N.
 # Пример:
-# - пусть N = 4, тогда [ 1, 2, 6, 24 ] (1, 1*2, 1*2*3, 1*2*3*4)
\ No newline at end of file
+# - пусть N = 4, тогда [ 1, 2, 6, 24 ] (1, 1*2, 1*2*3, 1*2*3*4)
+
+n = int(input("Введите целое число: "))
+
+
+def multiply_n(user_element):
+    temp, mult, result = 1, 1, []
+    while temp <= user_element:
+        mult *= temp
+        temp += 1
+        result.append(mult)
+
+    return result
+
+
+print(f'Набор произведений от 1 до {n} равен - {multiply_n(n)}')
Index: venv/lesson-2/task-8-additionally.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/venv/lesson-2/task-8-additionally.py b/venv/lesson-2/task-8-additionally.py
--- a/venv/lesson-2/task-8-additionally.py	
+++ b/venv/lesson-2/task-8-additionally.py	
@@ -1,3 +1,18 @@
 # Задайте список из n чисел последовательности $(1+\frac 1 n)^n$ и выведите на экран их сумму.
 # Пример:
-# - Для n = 6: {1: 4, 2: 7, 3: 10, 4: 13, 5: 16, 6: 19}
\ No newline at end of file
+# - Для n = 6: {1: 4, 2: 7, 3: 10, 4: 13, 5: 16, 6: 19}
+
+n = int(input("Введите целое число: "))
+
+
+def multiply_n(user_element):
+    i, mult, result = 1, 1, {}
+    while i <= user_element:
+        mult += round((1 + 1 / user_element)**user_element)
+        result[i] = mult
+        i += 1
+    return result
+
+
+print(f'Сумма последовательности чисел - {multiply_n(n)}')
+
